scylla/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  scylla-db:
+    image: scylladb/scylla:${SCYLLA_VERSION}
+    container_name: scylla-db
+    ports:
+      - "9100:9100"
+      - "9180:9180"
+      - "10000:10000"
```

Comment:
Do we use all these ports, or are some of them artifacts of the now-dropped Scylla Manager integration?

Reply:
We just use 9180, the others are there for convenience, but they can probably be removed since we don't use them with our tests.